CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+=======
+1.7.0 (20??-??-??)
+------------------
+
+  * Namespace changed from `org\bovigo\vfs` to `bovigo\vfs`
+    - The old namespace still works, but has been deprecated. It will be removed in version 2.
```

Comment:
Document the bump to 5.6 as a PHP minimum version

Reply:
Point added.